neo4j/v4.x.x/wrapper.sh

```diff
@@ -26,20 +26,20 @@ set -m
 /docker-entrypoint.sh neo4j &
 
 # Wait for Neo4j
-log_info "Waiting for Neo4j to start..."
-wget --quiet --tries=10 --waitretry=2 -O /dev/null http://localhost:7474
+log_info "Checking to see if Neo4j has started..."
+wget --quiet --tries=10 --waitretry=10 -O /dev/null http://localhost:7474
+log_info "Neo4j has started 🤓"
 
 # Import data
 log_info  "Loading and importing Cypher file(s)..."
 
 for cypherFile in /var/lib/neo4j/import/*.cypher; do
     log_info "Processing ${cypherFile}..."
     contents=$(cat ${cypherFile})
-    # cypher-shell -u ${NEO4J_USER} -p ${NEO4J_PASSWORD} -a bolt://localhost:7687 "${contents}"
-    cat ${cypherFile} | cypher-shell -u ${NEO4J_USER} -p ${NEO4J_PASSWORD} -a bolt://0.0.0.0:7687 --format plain
+    cat ${cypherFile} | cypher-shell -u ${NEO4J_USER} -p ${NEO4J_PASSWORD} --format plain
```

Comment:
Key change here - remove the `-a bolt://0.0.0.0:7687` portion. No need to specify a host to connect to - we're already running within the container that is our Neo4j server